bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
+    },
+    {
+      "id": 416,
+      "question": "Which newline sequence does HTTP use?",
+      "answer": "carriage return line feed, CRLF, \\r\\n"
+    },
+    {
+      "id": 417,
+      "question": "What does one call the optimization technique used in CPU design that attempts to guess the outcome of a conditional operation and prepare for the most likely result?",
+      "answer": "branch prediction"
+    },
+    {
+      "id": 418,
+      "question": "Name a universal logic gate.",
+      "answer": "NAND, NOR"
+    },
+    {
+      "id": 419,
+      "question": "What is the mathematical formalism which functional programming was built on?",
+      "answer": "lambda calculus"
+    },
+    {
+      "id": 420,
+      "question": "Why is a DDoS attack different from a DoS attack (answer in letters - A. because the victim's server was indefinitely disrupted from the amount of traffic, B. because it also attacks the victim's confidentiality, C. because the attack had political purposes behind it, D. because the traffic flooding the victim originated from many different sources)?",
+      "answer": "D"
+    },
+    {
+      "id": 421,
+      "question": "What is a HTTP/1.1 feature that was superseded by HTTP/2 multiplexing and is unsupported in most browsers nowadays?",
+      "answer": "pipelining"
+    },
+    {
+      "id": 422,
+      "question": "Which of these languages is the oldest? Tcl, Smalltalk 80, Haskell, Standard ML, Java",
+      "answer": "Smalltalk 80"
+    },
+    {
+      "id": 423,
+      "question": "What is the name for unicode codepoints that do not fit into 16 bits?",
+      "answer": "surrogates"
+    },
+    {
+      "id": 424,
+      "question": "Under what locale does making a string lowercase behave differently?",
+      "answer": "Turkish"
+    },
+    {
+      "id": 425,
+      "question": "What does the \"a\" represent in a HSLA color value?",
+      "answer": "transparency, translucency, alpha value, alpha channel"
+    },
+    {
+      "id": 426,
+      "question": "What is the section of a GIF that is limited to 256 colors called?",
+      "answer": "image block"
+    },
+    {
+      "id": 427,
+      "question": "What is an interpreter capable of interpreting itself called?",
+      "answer": "metainterpreter"
+    },
+    {
+      "id": 428,
+      "question": "Due to what data storage medium did old programming languages, such as cobol, ignore all characters past the 72nd column?",
+      "answer": "punch cards"
+    },
+    {
+      "id": 429,
+      "question": "Which of these sorting algorithms is not stable? Counting sort, quick sort, insertion sort, tim sort, bubble sort",
+      "answer": "quick, quick sort"
+    },
+    {
+      "id": 430,
+      "question": "Which of these languages is the youngest? Lisp, Python, Java, Haskell, Prolog, Ruby, Perl",
+      "answer": "Java"
+    }
+  ],
+  "python": [
+    {
+      "id": 501,
+      "question": "Is it all objects (y/n)?",
```

Comment:
I see that there's a comment marked as resolved here that asks if it's necessary, which was just answered with 'yes'. While I'm personally unsure whether this should be a question, at the very least I think it's badly phrased, and will lead to confusion, a more suitable question I feel would be along the lines of "Is everything in Python an object?", since "Is it all objects?" sounds too vague.

Reply:
i agree with the last part, the question should be phrased better.